README.txt
This extension is just an umbrella for a suite of extensions. It provides a general manual about it (see doc/manual.sxw). It is also used to simplify installation by providing suggestions to the most important extensions composing this suite.

ChangeLog
2013-11-06 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Removed old require_once() calls, resolves #53386

2013-06-21 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Released to TER as version 1.7.0

2013-05-22 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Added template retrieval utility (FAL support), resolves #48470

2013-04-22 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Verified compatibility with TYPO3 CMS 6.1, resolves #47453

2013-04-11 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Added API for getting controller properties, resolves #38398

2013-04-01 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Verified compatibility with TYPO3 CMS 6.0, resolves #46842

2012-07-30 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Released to TER as version 1.6.0

2012-07-23 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Added new "engine" property to Data Filter structure and base class, resolves #39190

2012-07-10 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Released to TER as version 1.5.0

2012-07-07 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Added check for missing service, resolves #38739

2012-06-26 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Cleaned up some code, verified compatibility with TYPO3 4.7, raised requirements to TYPO3 4.5, resolves #38400

2012-04-19 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Released to TER as version 1.4.0

2012-04-03 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Removed obsolete method tx_tesseract_utilities::wrapMessage(), resolves #32280

2012-03-20 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Moved initEmptyDataStrucure() method to data provider base class instead of interface, resolves #35048
	* Corrected creation of empty data structure to take structure type into account, resolves #35051

2012-02-06 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Released to TER as version 1.3.0

2012-01-19 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Added documentation about new negate flag for filter conditions, resolves #33322

2012-01-12 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Corrected description of tx_tesseract_datacontroller_output::getPrefixId(), resolves #33154
	* Added API for message queue handling in the controller, resolves #9072

2012-01-08 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Implemented improvements to the API, resolves #33042

2011-12-03 Francois Suter (Cobweb)  <typo3@cobweb.ch>

    * Added utility method for getting a language object, resolves #32281

2011-08-16 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Released to TER as version 1.2.0

2011-07-18 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Added API for empty filter check, resolves #28285

2011-07-17 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Added documentation about void filters, resolves #28276

2011-05-18 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Released to TER as version 1.1.0

2011-05-17 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Enabled use of TypoScript to define default values for components properties, resolves #13660

2011-03-22 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Updated documentation for inclusion of filter information in data structure, resolves #13972
	* Updated documentation to describe newer Tesseract components
	* Released to TER as version 1.0.2

2010-12-28 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Released to TER as version 1.0.1

2010-12-20 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Data filter information should contain original filter information, resolves #11599
	* Added warning about extensions installation order, resolves #11549

2010-11-15 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Improved wording of installation check screen, resolves #10162

2010-10-05 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* Added data setter method to Data Filter interface, resolves #10059

2010-09-27 Francois Suter (Cobweb)  <typo3@cobweb.ch>

	* First public release


ext_emconf.php
<?php

/*********************************************************************
* Extension configuration file for ext "tesseract".
*
* Generated by ext 21-06-2013 12:49 UTC
*
* https://github.com/t3elmar/Ext
*********************************************************************/

$EM_CONF[$_EXTKEY] = array (
  'title' => 'Project Tesseract',
  'description' => 'This projects covers a suite of extensions designed to facilitate the extraction of data and their display, using controllers and standard data formats. More info on http://www.typo3-tesseract.com/',
  'category' => 'misc',
  'author' => 'Francois Suter (Cobweb)',
  'author_email' => 'typo3@cobweb.ch',
  'shy' => '',
  'dependencies' => 'expressions',
  'conflicts' => '',
  'priority' => '',
  'module' => '',
  'state' => 'stable',
  'internal' => '',
  'uploadfolder' => 0,
  'createDirs' => '',
  'modify_tables' => '',
  'clearCacheOnLoad' => 0,
  'lockType' => '',
  'author_company' => '',
  'version' => '1.7.0',
  'constraints' => 
  array (
    'depends' => 
    array (
      'php' => '5.2.0-0.0.0',
      'typo3' => '4.5.0-6.1.99',
      'expressions' => '',
    ),
    'conflicts' => 
    array (
    ),
    'suggests' => 
    array (
    ),
  ),
  '_md5_values_when_last_written' => 'a:27:{s:9:"ChangeLog";s:4:"19a7";s:22:"class.tx_tesseract.php";s:4:"4ba3";s:35:"class.tx_tesseract_emconfhelper.php";s:4:"1449";s:16:"ext_autoload.php";s:4:"5b7d";s:21:"ext_conf_template.txt";s:4:"464c";s:12:"ext_icon.gif";s:4:"37ae";s:17:"ext_localconf.php";s:4:"6394";s:14:"ext_tables.php";s:4:"f8c0";s:13:"locallang.xml";s:4:"554c";s:10:"README.txt";s:4:"a21e";s:44:"base/class.tx_tesseract_picontrollerbase.php";s:4:"975c";s:14:"doc/manual.pdf";s:4:"6cac";s:14:"doc/manual.sxw";s:4:"0046";s:14:"doc/manual.txt";s:4:"07a5";s:50:"interfaces/interface.tx_tesseract_dataconsumer.php";s:4:"4125";s:52:"interfaces/interface.tx_tesseract_datacontroller.php";s:4:"deea";s:59:"interfaces/interface.tx_tesseract_datacontroller_output.php";s:4:"2898";s:48:"interfaces/interface.tx_tesseract_datafilter.php";s:4:"b808";s:50:"interfaces/interface.tx_tesseract_dataprovider.php";s:4:"4ff3";s:36:"lib/class.tx_tesseract_exception.php";s:4:"5ec0";s:36:"lib/class.tx_tesseract_utilities.php";s:4:"f8e3";s:41:"services/class.tx_tesseract_component.php";s:4:"03d5";s:44:"services/class.tx_tesseract_consumerbase.php";s:4:"8cf1";s:46:"services/class.tx_tesseract_controllerbase.php";s:4:"ed24";s:46:"services/class.tx_tesseract_feconsumerbase.php";s:4:"d948";s:42:"services/class.tx_tesseract_filterbase.php";s:4:"3952";s:44:"services/class.tx_tesseract_providerbase.php";s:4:"0a52";}',
  'suggests' => 
  array (
  ),
  'comment' => 'Compatibility with TYPO3 6.0 and 6.1; improved API for getting controller properties.',
  'user' => 'francois',
);

?>